app/helpers/Emails/Notifications/Assignments/newShadowAssignmentEmail_en.latte

```diff
@@ -1,3 +1,4 @@
+{emailSubject}New shadow assignment - {$assignment}{/emailSubject}
```

Comment:
All other subjects are capitalized.

Reply:
Looks like some kind of leftovers... fixed